index.html

```diff
@@ -167,13 +167,23 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
                 </div>
                 <div class="col-lg-6 col-md-6 col-sm-12 col-12">
                     <form action="">
-                        <div class="input-group">
+                        <!-- <div class="input-group">
```

Comment:
You can remove the unnecessary code no need to keep comments.

[Yagni](https://martinfowler.com/bliki/Yagni.html)